cmd/kubelet/app/options/options.go

```diff
@@ -310,9 +305,6 @@ func (f *KubeletFlags) AddFlags(fs *pflag.FlagSet) {
 	f.ContainerRuntimeOptions.AddFlags(fs)
 
 	fs.Var(&f.KubeConfig, "kubeconfig", "Path to a kubeconfig file, specifying how to connect to the API server.")
```

Comment:
Please add the following to the end of the help text: `Providing --kubeconfig enables API server mode, omitting --kubeconfig enables standalone mode.` Please also annotate the field in the KubeletFlags struct with a comment that matches the help text.